src/texdocdialog.cpp

```diff
@@ -39,20 +39,20 @@ TexdocDialog::~TexdocDialog()
  * \brief generate all rows for table
  * \param all filter to used packages or all ctan packages
  */
-void TexdocDialog::regenerateTable(int state)
+void TexdocDialog::regenerateTable(bool cbAllPackagesChecked)

```

Comment:
this kind of works because c++ tends to interpret integer>0 as bool, but the connected signal uses integer 0,1 or 2. (pratically 0 and 2)

So, I would suggest to limit the changes to what is actually reported to be changed which is lines 185..188

Reply:
done